frontend/src/components/articles/ArticleItem.vue

```diff
@@ -18,22 +18,32 @@ const emit = defineEmits(["reloadArticleView"]);
 
 const props = defineProps<{
   article: Article;
+  isNew?: boolean;
 }>();
+
 const confirm = useConfirm();
+const notifications = useNotifications();
+
 const articlesType = Object.values(ArticleKind);
 
 const editableArticle = ref(props.article);
-const notifications = useNotifications();
 
 const onUpdateArticle = async () => {
-  await updateArticle(`${editableArticle.value.id}`, editableArticle.value);
-  notifications.showUpdateArticleNotification();
+  if (props.isNew) {
```

Comment:
Can we highlight the new article somehow visually to make clear this is a new item? Maybe we can add a slightly darker background color and/or add a title to the card? 🤔